apisix/cli/schema.lua

```diff
@@ -259,6 +259,14 @@ local config_schema = {
                     default = false,
                     description = "a global switch to disable upstream health checks",
                 },
+                trusted_addresses = {
+                    type = "array",
+                    minItems = 1,
+                    items = {
+                        type = "string",
```

Comment:
"oneOf": [
            {"format": "ipv4"},
            {"format": "ipv6"}
        ]

Reply:
There are a few problems with doing this:

1. Need CIDR support
2. As discussed in https://github.com/apache/apisix/pull/12551#discussion_r2347756462, try to use the existing schema_def. A feasible solution is to copy the corresponding schema again.